src/util/workerthread.h

```diff
@@ -134,6 +135,7 @@ class WorkerThread : public QThread {
 
   private:
     const QString m_name;
+    QThread::Priority m_priority;
```

Comment:
Does not need to be a member, the constructor parameter is sufficient.

Reply:
I'm not so sure if the priority can be set in the constructor or if the constructor is still in the parent thread. I therefor moved the setPriority logic to be called in the run method which is definitely inside the target thread. So, variable is necessary afaik.

Reply:
That's correct. I misread the diff and didn't notice that the priority is needed later in run(). Then please declare it as `const`.

Reply:
done